ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_logic.py

```diff
@@ -724,3 +724,34 @@ def test_jax_numpy_isinf(
         on_device=on_device,
         x=x[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.numpy.isfinite",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"), allow_inf=True
+    ),
+)
+def test_jax_numpy_isfinite(
+        *,
+        dtype_and_input,
+        as_variable,
+        with_out,
+        num_positional_args,
+        native_array,
+        on_device,
+        fn_tree,
+        frontend,
+):
+    input_dtype, input = dtype_and_input
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=input[0],
```

Comment:
This should be `x=x[0]` given that your frontend implementation uses x as variable name